CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 ## Unreleased
  - TBD
 
+## 10.1.0
+ - [Fix exports and prepare release of erdjs 10](https://github.com/ElrondNetwork/elrond-sdk-erdjs/pull/197)
+
 ## 10.0.1-beta.2
```

Comment:
I would remove the beta changelog and put everything in 10.1.0

Reply:
Makes sense. Will fix.